docs/src/main/asciidoc/security-openid-connect-client-reference.adoc

```diff
@@ -1117,6 +1117,42 @@ The `quarkus-oidc-token-propagation-reactive` extension provides `io.quarkus.oid
 The `quarkus-oidc-token-propagation-reactive` extension (as opposed to the non-reactive `quarkus-oidc-token-propagation` extension) does not currently support the exchanging or resigning the tokens before the propagation.
 However, these features may be added in the future.
 
+[[oidc-client-graphql-client]]
+== GraphQL client integration
+
+The `quarkus-oidc-client-graphql` extension provides a way to integrate an
+OIDC client into GraphQL clients. This works similarly as with REST clients.
+When this extension is present, any configured (that means NOT created
+programmatically via the builder, but via configuration properties) GraphQL
+client will attempt to use the OIDC client to obtain the value of the
+`Authorization` header.
+
+To configure which OIDC client should be used by GraphQL clients, use
+
+----
+quarkus.oidc-client-graphql.client-name=OIDC_CLIENT_NAME
```

Comment:
How will it work if no named, default OIDC client is configured only ? FYI, the default client name is `DEFAULT` so if it also has to be referred to by name then it should be mentioned in a `NOTE`

Reply:
Added a note that if undefined, it will attempt to use the default OIDC client